javascript/javascript2/week1/sprit-animal.js

```diff
@@ -0,0 +1,29 @@
+const button = document.getElementById("myBtn");
+const input = document.getElementById("name");
+const p = document.querySelector("p");
+const animals = [
+    "The strong Bear",
+    "The patient Cat",
+    "The joker Coyote",
+    "The fearless Crow",
+    "The innocent Deer",
+    "The protective Dolphin",
+    "The joyfull Dragonfly",
+    "The angery Lion",
+    "The peacefull Panda",
+    "The driving force Horse",
+    "The creative Spider",
+    "The humble Turtle",
+];
+const users = [];
+button.addEventListener("click", function(event) {
+    const randomNumber = Math.floor(Math.random() * 12);
```

Comment:
Don't put random constants in your code.
```suggestion
    const randomNumber = Math.floor(Math.random() * animals.length);
```

Reply:
fixed